client/layout/index.jsx

```diff
@@ -381,6 +382,8 @@ class Layout extends Component {
 			// There is a custom command palette in the "Switch site" page, so we disable it.
 			config.isEnabled( 'yolo/command-palette' ) && this.props.currentRoute !== '/switch-site';
 
+		const isA4AClient = this.props.sectionName === 'a8c-for-agencies-client';
```

Comment:
I guess we don't need a feature flag here, do we?

Reply:
We want the client details to be fetched only for some specific URLs, not always.